purchases/src/defaults/kotlin/com/revenuecat/purchases/listenerConversions.kt

```diff
@@ -240,6 +255,16 @@ fun Purchases.getAmazonLWAConsentStatusWith(
     getAmazonLWAConsentStatus(getAmazonLWAConsentStatusListener(onSuccess, onError))
 }
 
+/**
+ * Gets the [CustomerCenterConfigData] for the current user. Used by RevenueCatUI to present the CustomerCenter.
+ */
+fun Purchases.getCustomerCenterConfigDataWith(
```

Comment:
This is meant to be used internally, so maybe it's not needed to create these listener and async versions of this API... But I thought I might as well add them for consistency. Wdyt?

Reply:
Or we could make `Purchases.getCustomerCenterConfigData` exactly how we want it (e.g. suspend or function parameters), as we don't need to care about Java compatibility since `RevenueCatUI` is Kotlin.

Reply:
Hmm right... FWIW, another reason to make these public is in case devs want to use the customer center config data themselves (same with paywall data within `Offering`)... But yeah, this seems like a less likely case for this to be useful... Will remove those and we can always add those later if needed 👍